src/content/getting-started/prepare-your-provider-account/vmware-cloud-director/_index.md

```diff
@@ -35,7 +35,7 @@ In terms of virtual networking, the following requirements must be met:
 
 - The virtual machines (VMs) must have access to the VCD API endpoint for the controllers to work.
 
-- Access to the Internet is required for some of our automation to pull artifacts from the registry or get new certificates from Let's Encrypt. It's possible to configure an egress HTTP proxy to control the traffic. You can check the list of domains that need to be accessible [here]({{< relref "/vintage/platform-overview/security/cluster-security/domain-allowlist/#on-premise-installations" >}}).
+- Access to the Internet is required for some of our automation to pull artifacts from the registry or get new certificates from Let's Encrypt. It's possible to configure an egress HTTP proxy to control the traffic. You can check the list of domains that need to be accessible [here]({{< relref "/vintage/platform-overview/security/cluster-security/domain-allowlist/index.md#on-premise-installations" >}}).
```

Comment:
Not sure if we should add this change to the scope of this PR? Was that by accident?

Reply:
I can remove this though

Reply:
Naa well, then let's keep it in, it's not a big change so whatever :D